src/sections/Community/Handbook/github-process.js

```diff
@@ -0,0 +1,463 @@
+import React from "react";
+import { Container } from "../../../reusecore/Layout";
+import { HandbookWrapper } from "./Handbook.style";
+import TOC from "../../../components/handbook-navigation/index";
+import TocPagination from "../../../components/handbook-navigation/TocPagination";
+
+const SecurityVulnerabilitiesPage = () => {
+  return (
+    <HandbookWrapper>
+      <div className="page-header-section">
+        <h1>GitHub Process</h1>
+      </div>
+      <TOC />
+      <div className="page-section">
+        <Container>
+          <div className="content">
+            <h2>Issue and PR Labels and Templates, Bots</h2><br/>
+            <h3><strong>Overview</strong></h3><br />
+            <p>The Layer5 GitHub Process emphasizes consistent pull requests and issue labels to streamline continuous integration and reduce overhead across the growing repository collection.</p>
+            <p>The number of repositories under the “Layer5io” organization is growing. In order to ensure consistency across the repositories and to decrease the time it takes for a continuous integration (GitHub Actions) to execute, consistent use of pull request and issue labels will aid in the reduction of overhead.</p>
```

Comment:
Right on. Also, worth noting: the Layer5 community spans five GitHub organizations - https://layer5.io/community/handbook/repository-overview